test/Microsoft.AspNetCore.JsonPatch.Test/Dynamic/RemoveOperationTests.cs

```diff
@@ -80,7 +80,7 @@ public void RemovePropertyFromExpandoObject()
         }
 
         [Fact]
-        public void RemovePropertyFromExpandoObjectMixedCase()
+        public void RemovePropertyFromExpandoObjectMixedCaseThrowsPathNotFoundException()
```

Comment:
You can break up these names with underscores to make it more readable. 

`RemoveProperty_FromExpandoObject_MixedCase_ThrowsPathNotFoundException`